frontend/packages/console-app/src/components/tour/GuidedTour.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { TourContext } from './tour-context';
+import TourCloseModal from './TourCloseModal';
+import StepComponent from './StepComponent';
+
+const GuidedTour: React.FC = () => {
+  const { tourState, tour, totalSteps, onComplete } = React.useContext(TourContext);
+  if (!tour) return null;
+  const { intro, steps, end } = tour;
+  const { stepNumber, closeTour, startTour, completedTour } = tourState;
+  if (completedTour) {
+    onComplete();
+    return null;
+  }
+  if (closeTour) return <TourCloseModal />;
+  if (startTour || stepNumber === 0)
+    return (
+      <StepComponent
+        {...intro}
```

Comment:
```suggestion
      <StepComponent
        showClose={false}
        {...intro}
```

Waiting on confirmation from UX but I think we should hide the close button for the intro and outro modals.